src/flex.skl

```diff
@@ -1994,20 +1996,22 @@ void yyFlexLexer::yyrestart( std::istream* input_file )
 	yy_load_buffer_state( M4_YY_CALL_ONLY_ARG );
 
 m4_ifdef( [[M4_YY_USES_REJECT]], [[        
-        /* Ensure the reject state buffer is large enough.
-         */
-        if ( YY_G(yy_state_buf_max) < YY_CURRENT_BUFFER_LVALUE->yy_buf_size ) {
-          new_size = YY_G(yy_state_buf_max) + YY_CURRENT_BUFFER_LVALUE->yy_buf_size;
-          new_state_buf = (yy_state_type*) yyrealloc( YY_G(yy_state_buf), new_size * sizeof(yy_state_type) M4_YY_CALL_LAST_ARG );
-          
-          if ( new_state_buf == NULL ) {
-            YY_FATAL_ERROR( "out of dynamic memory in yylex()" );
-          }
-          else {
-            YY_G(yy_state_buf) = new_state_buf;
-            YY_G(yy_state_buf_max) = new_size;
-          }
-        }
+		/* Ensure the reject state buffer is large enough.
+		 */
+		if ( YY_G(yy_state_buf_max) < (YY_CURRENT_BUFFER_LVALUE->yy_buf_size + YY_STATE_BUF_EXTRA_SPACE) ) {
+			/* yy_state_buf_max already contains YY_STATE_BUF_EXTRA_SPACE */
+			/* new_size = YY_G(yy_state_buf_max) + YY_CURRENT_BUFFER_LVALUE->yy_buf_size; */
```

Comment:
This line of commented code looks like it was meant to be removed.

Reply:
Yep, thanks for catching that. Fixed it.